src/aggregator/aggregator/elem_base.go

```diff
@@ -105,6 +105,9 @@ const (
 	forwardTypeRemote
 )
 
+// keep in sync with type list above
+const numForwardTypes = 2
```

Comment:
Hmm probably no way we can verify that invariant in tests, is there? That said it's close enough to the other one that it's probably ok. Maybe worth adding a note to the effect of "This is important because we directly convert ints to the enum type"?

Reply:
I'll update comment, but re: testing I think it's sufficient to rely on other unit tests indirectly catching this, as it'll hard-panic even without a valid Tally scope. I don't know how else we can enforce this without reflection magic.
Or we can set the `forwardType` const last value to be `forwardTypeInvalid` and use that as sentinel value (and leave a comment reflecting that).

Reply:
Yeah no strong disagreements here (though fwiw I am for reflection magic in tests where it's not too magic and provides otherwise difficult to enforce guarantees). Can't think of an easy way to test this case though so works for me!